### PR TITLE
docs: improve readme to ease deploymentrun

### LINKED ISSUE #146: docs: Improve README
- [x] Steps to deploy the testnet should be clear and working, including the batcher
- [x] Steps to send a proof using the batcher  in Devnet with each proving system should be clear
- [x] Steps to send a proof without the batcher in Devnet with each proving system should be clear
- [x] Steps to send a proof using the batcher  in Testnet with each proving system should be clear
- [x] Steps to send a proof without the batcher in Testnet with each proving system should be clear

### PR DESCRIPTION
**Pull request by JuArce**
- New README_NEW.md file
- Complete makefile help section

### CODE REVIEW

File: README_NEW.md [hunk 1]
```diff
@@ -0,0 +1,654 @@
...
+
+#### Holesky/Mainnet
+
+EigenLayer strategies are available in [eigenlayer-strategies](https://holesky.eigenlayer.xyz/restake).
+
+For Holesky, we are using [WETH](https://holesky.eigenlayer.xyz/restake/WETH) as the strategy token.
+
+To obtain HolETH and swap it for different strategies, you can use the following [guide](https://docs.eigenlayer.xyz/eigenlayer/restaking-guides/restaking-user-guide/stage-2-testnet/obtaining-testnet-eth-and-liquid-staking-tokens-lsts).
+
+### Register into AlignedLayer
```
Comment (glpecile) [inline README_NEW.md]: replace AlignedLayer for Aligned

File: README_NEW.md [hunk 2]
```diff
@@ -0,0 +1,654 @@
...
+
+To register an operator in AlignedLayer run the following command:
```
Comment (glpecile) [inline README_NEW.md]: replace AlignedLayer for Aligned

File: README_NEW.md [hunk 3]
```diff
@@ -0,0 +1,654 @@
...
+make bindings
+```
+
+
+## Run Devnet Locally with Anvil
+
+Start anvil with every relevant contract deployed with:
+
+```bash
+make anvil_start
```
Comment (MauroToscano) [inline README_NEW.md]:
```suggestion
make anvil_start_with_block_time
```

File: README_NEW.md [hunk 4]
```diff
@@ -0,0 +1,654 @@
...
+  private_key_store_password: <bls_private_key_store_password>
+
+## EigenDA Configurations
+eigen_da_disperser:
+  url: <eigen_da_disperser_url> # This is the url of the EigenDA Disperser
+
+## Celestia Configurations
+celestia:
+  url: <celestia_url> # This is the url of the deployed Celestia Light node
+  keystore: <celestia_keystore> # This is the keystore of the Celestia Light node
```
Comment (MauroToscano) [inline README_NEW.md]: Remove this

File: README_NEW.md (hunk 1 above)
Comment (NicolasRampoldi) [reply README_NEW.md]: Solved in [9e80cac](https://github.com/yetanotherco/aligned_layer/pull/152/commits/9e80cac2c185ef42fd5c9c6439b43a9eab385cee)

File: README_NEW.md (hunk 2 above)
Comment (NicolasRampoldi) [reply README_NEW.md]: Solved in [9e80cac](https://github.com/yetanotherco/aligned_layer/pull/152/commits/9e80cac2c185ef42fd5c9c6439b43a9eab385cee)

File: README_NEW.md (hunk 3 above)
Comment (NicolasRampoldi) [reply README_NEW.md]: Solved in [1c49ca6](https://github.com/yetanotherco/aligned_layer/pull/152/commits/1c49ca67a90eed08d341073d87e60ff4453c85bb)

File: README_NEW.md (hunk 4 above)
Comment (NicolasRampoldi) [reply README_NEW.md]: Solved in [c8752dd](https://github.com/yetanotherco/aligned_layer/pull/152/commits/c8752ddbefedbcb4a80d4847468e26047b780a65)

**Review by MauroToscano — APPROVED** [review]

**Review by entropidelic — APPROVED** [review]

**Review by MauroToscano — APPROVED** [review]

**Pull request merged**